### PR TITLE
design: implement mission statement in footer (WIP)

### LINKED ISSUE #501: Mission Statement in Footer einbinden
Text: 
Serlo.org ist die Wikipedia fürs Lernen. Wir sind eine engagierte Gemeinschaft die daran arbeitet, hochwertige Bildung weltweit frei verfügbar zu machen. Unterstütze uns dabei mit einer Spende oder mach mit!

Anmerkungen:
"Spende" --> bold, verlinkt auf www.serlo.org/spenden
"mach mit" --> bold, verlinkt auf www.serlo.org/mitmachen

### PR DESCRIPTION
**Pull request by Knorrke**
closes #501
Please take a look, @arekkas, @inyono, @elbotho

Things to be done before merging:
- [x] add translations for mission statement
- [x] add new navigation "subject-nav" in database

Things to be done after merging:
- [ ] update serlo.config.local.php to the correct slogan on server
- [ ] populate navigations with correct links

### CODE REVIEW

File: src/assets/source/images/home_donate.svg [hunk 1]
```diff
@@ -4,20 +4,20 @@
 <svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
 	 width="90px" height="90px" viewBox="0 0 90 90" enable-background="new 0 0 90 90" xml:space="preserve">
 <g>
-	<path fill="#95BC1A" d="M66.436,39.55c0-6.061-4.913-10.977-10.977-10.977c-5.309,0-9.736,3.771-10.755,8.781
+	<path fill="#fff" d="M66.436,39.55c0-6.061-4.913-10.977-10.977-10.977c-5.309,0-9.736,3.771-10.755,8.781
```
Comment (Knorrke) [inline src/assets/source/images/home_donate.svg]: oh sorry, this quick-fix should not be in the pull request. I'll change this

**Comment by Knorrke** [comment]
I will post screenshots as soon as my internet connection is stable enough :D

**Comment by Knorrke** [comment]
Desktop:
desktop

Tablet:
tablet

Mobile:
mobile

**Comment by elbotho** [comment]
still not completely happy with the "stay in touch"-part, but rest is great.
Maybe I will make minor improvements after it is online.
Thank you @Knorrke !

**Comment by Knorrke** [comment]
in my current implementation the type "subject-nav" for the central footer navigation has to be added to the database manually by the following comand: 
```
INSERT INTO type (name) VALUES ('subject-nav')
```

**Comment by arekkas** [comment]
@Knorrke are the todos still todos or is this good to go? I ran the SQL command in prod

**Comment by Knorrke** [comment]
Translations still have to be done, @inyono will add them since I couldn't set up poedit correctly. So I'd wait for this before merging.


As of the other todos:
I will start populating the navigation, that's pretty independent of the PR. The config file has to be changed after merge.

**Comment by inyono** [comment]
Translations done

**Comment by arekkas** [comment]
Nice! What else needs to be done?

**Comment by Knorrke** [comment]
Thats it :) Good for reviewing.
But after deployment the php config has to be changed and the subject-navigation has to be set up, thats the remaining two tasks in the first post.

**Comment by arekkas** [comment]
Ok, the navigation can be done through the backend, right? Do you want to update that?

**Comment by Knorrke** [comment]
Yes, exactly. And its only the new `subject-nav` navigation, the general and legal terms are already correct.
I can do that, yes.

**Comment by arekkas** [comment]
I have to delay this merge because of how instance templates work in GCP - aka it will not be possible to roll back the build toolchain PR if I merge this.

**Pull request merged**